/*
 * Copyright (c) 2016 Cisco and/or its affiliates, and 
 *                    Cable Television Laboratories, Inc. ("CableLabs")
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at:
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

 /*
  * File has been generated by proto_generator.py
  */

syntax = "proto2";


message t_LED {
    enum t_ledType {
        LED_TYPE_US2_RF      = 1;
        LED_TYPE_ALARM       = 2;
        LED_TYPE_STATUS      = 3;
        LED_TYPE_DAISY_CHAIN = 4;
        LED_TYPE_US1_RF      = 5;
        LED_TYPE_DS_RF       = 6;
        LED_TYPE_SFP2        = 7;
        LED_TYPE_SFP1        = 8;
        LED_TYPE_REPLACEME   = 9;
    }
    enum t_ledColor {
        LED_COLOR_GREEN      = 1;
        LED_COLOR_YELLOW     = 2;
    }
    enum t_ledAction {
        LED_ACTION_LIT       = 1;
        LED_ACTION_FLASHING  = 2;
        LED_ACTION_DARK      = 3;
    }
    message t_ledCfg {
        optional t_ledType   ledType = 1;
        optional t_ledColor  color   = 2;
        optional t_ledAction action  = 3;
    }
    optional t_ledCfg setLed = 1;
}
